sqlglot/dialects/bigquery.py

```diff
@@ -168,12 +168,18 @@ def _pushdown_cte_column_names(expression: exp.Expression) -> exp.Expression:
     return expression
 
 
-def _parse_timestamp(args: t.List) -> exp.StrToTime:
+def _parse_parse_timestamp(args: t.List) -> exp.StrToTime:
```

Comment:
why is it called parse parse

Reply:
The function we're constructing is called `PARSE_TIMESTAMP`, so that's where the suffix comes from. The `_parse` prefix is to be consistent with other helpers in the file (and in general).

Imho "parse" is the wrong term to use for `FUNCTIONS` entries - a better fit would be something like "construct" which reflects what we're doing. What do you think?

Reply:
sure you can change it to build

Reply:
let's do in a followup PR, I'll make sure to clean up other similar function names

Reply:
sure

Reply:
we can just leave it as is